src/main/java/org/jitsi/jigasi/GatewaySession.java

```diff
@@ -58,6 +58,21 @@
         = "JITSI_MEET_ROOM_PASS_HEADER_NAME";
 
     /**
+     * Property name of optional INVITE header which specifies the base domain
+     * to be used to extract the subdomain from the roomname in order
+     * to construct custom bosh URL to enter MUC room that is hosting
+     * the Jitsi Meet conference.
+     */
+    public String JITSI_MEET_DOMAIN_BASE_HEADER
```

Comment:
It seems strange to use a field as constant and a default value at the same time.